src/main/java/org/ossgang/commons/observable/Observables.java

```diff
@@ -16,7 +22,7 @@ private Observables() {
      * subscribes to the upstream observable and caches its latest value.
      *
      * @param observable the observable
-     * @param <T> the type
+     * @param <T>        the type
```

Comment:
Did you switch formatter? Do we really like these empty spaces ?

Reply:
Not willingly haha I now use IntelliJ 2019.2.1 and this is the default.. I don't care too much, whatever you decide

Reply:
I think I would prefer the old one (of course one can argue if this is just because I am used to it) ... still I find it better readable .... I think that I remember even a chapter in Clean code which went into this direction ... I will try to look it up ;-)

Reply:
found it ... p87, "Horizontal Alignment" ;-)